### PR TITLE
[qwtw] new version 2.13.2 (spectrogramm functionality improved)

### PR DESCRIPTION
**Pull request by ig-or**
a few useful improvements and features for qwtw library

### CODE REVIEW

File: Q/qwtw/build_tarballs.jl [hunk 1]
```diff
@@ -52,4 +52,4 @@ dependencies = [
 
 # Build the tarballs, and possibly a `build.jl` as well.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               julia_compat="1.6", preferred_gcc_version = v"9.1.0")
+               julia_compat="1.8", preferred_gcc_version = v"9.1.0")
```
Comment (giordano) [inline Q/qwtw/build_tarballs.jl:55]: Any specific reason for this change?

**Review by giordano — COMMENTED** [review]
@barche this is using Qt_jll, should it be doing something else nowadays?

File: Q/qwtw/build_tarballs.jl (hunk 1 above)
Comment (ig-or) [reply Q/qwtw/build_tarballs.jl:55]: no any specific reason for this. The only change that might be important here is a move from C++14 into C++20..

**Comment by imciner2** [comment]
Why does this build script do `apk add g++`? We provide the g++ version alongside gcc, so this shouldn't be done (and CMake actually is ignoring the system version anyway, so this is just extraneous).

**Comment by ig-or** [comment]
Without those apk add g++, it didn't worked.. More details: 
[]([](https://github.com/JuliaPackaging/Yggdrasil/pull/2134/commits/9eaa610ea83a91633f4b71b136b4d934934e51aa))

**Comment by ig-or** [comment]
@giordano , @barche , do you think we can merge it ?

File: Q/qwtw/build_tarballs.jl (hunk 1 above)
Comment (fingolfin) [reply Q/qwtw/build_tarballs.jl:55]: If there is no reason to change the julia_compat, then maybe just don't change it?

Comment (fingolfin) [reply Q/qwtw/build_tarballs.jl:55]:
```suggestion
               julia_compat="1.6", preferred_gcc_version = v"9.1.0")
```

**Pull request merged**